includes/class-subscription-lists.php

```diff
@@ -462,13 +462,21 @@ public static function get_or_create_remote_list( $list ) {
 			throw new \Exception( 'Invalid list' );
 		}
 
-		$saved = Subscription_List::from_form_id( $list['id'] );
-
-		if ( $saved ) {
-			return $saved;
+		$subscriber_count = ! empty( $list['subscriber_count'] ) ? (int) $list['subscriber_count'] : 0;
+		$subscriber_count = ! empty( $list['member_count'] ) ? (int) $list['member_count'] : 0; // Tags have member_count instead of subscriber_count.
```

Comment:
@dkoo Isn't this setting the value of `$subscriber_count` even when only `subscriber_count` is present?

Shouldn't it be

```
$subscriber_count = ! empty( $list['member_count'] ) ? (int) $list['member_count'] : $subscriber_count;
```

?

Reply:
Ah yes, you're right—funny how no one has caught this since April!

Reply:
Strangely, it doesn't seem to affect the counts shown in my editor

![Uploading Screenshot 2024-06-24 at 5.37.56 PM.png…]()